ACE/ace/Time_Value.h

```diff
@@ -67,6 +67,9 @@ class ACE_Export ACE_Time_Value
   /// Constructor.
   explicit ACE_Time_Value (time_t sec, suseconds_t usec = 0);
 
+  /// Assign @ tv to this
+  ACE_Time_Value (const ACE_Time_Value &tv);
```

Comment:
Why does the default version not work? When there is a use-case where the default doesn't work we should add a test case for that